VERSIONING.md

```diff
@@ -0,0 +1,43 @@
+Versioning
+==============
+
+This document summarizes the versioning policy of releases in this github repository and is a variant of [Semantic Versioning](https://semver.org/). Within the core developer team, we have emphasized several times that it is important to keep releases backwards
+compatible. But this is an ideal plan. In the real world it is hard to achieve 100% backward compatibility.
```

Comment:
Waarom deze nuance?

Reply:
Because it is not a 100% guarantee, but we aim to be backwards compatible

Reply:
Waarom garanderen we dit niet dan?